src/fmu/dataio/datastructure/meta/content.py

```diff
@@ -181,200 +178,400 @@ class BoundingBox3D(BoundingBox2D):
 
 
 class Content(BaseModel):
```

Comment:
I would really like another name on this class than `Content`, as it also has a key (one of many) named `content`, and the class name `Content` is not very precise. Like `DataObject()` or better...

Reply:
I agree. Maybe we can go fully to just `Data`? It'd be much easier to understand if the object names mirror the block names as they appear in the schema, i.e. `data: Data`